src/components/Drawer/Drawer.style.ts

```diff
@@ -0,0 +1,61 @@
+import { DialogDisclosure as ReakitDialogDisclosure } from 'reakit';
+import styled from 'styled-components';
+
+import tokens from '../../tokens';
+
+export const DrawerDisclosure = ReakitDialogDisclosure;
+
+export const Drawer = styled.div.attrs({
+	className: 'c-drawer',
+})`
+	--c-drawer--color: var(--drawer--color, ${({ theme }) => theme.colors.textColor});
+	--c-drawer--background: var(--drawer--background, ${({ theme }) => theme.colors.modalBackground});
+	--c-drawer--heading--background: var(
+		--drawer--heading--background,
+		${({ theme }) => theme.colors.modalHeadingBackground}
+	);
+	--c-drawer--heading--border-color: var(
+		--drawer-heading--border-color,
+		${({ theme }) => theme.colors.modalHeadingBorderColor}
+	);
+
+	position: absolute;
+	flex-shrink: 0;
+	width: 30vw;
+	display: flex;
+	flex-direction: column;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	color: var(--c-drawer--color);
+	background: var(--c-drawer--background);
+	box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.25);
```

Comment:
```suggestion
	box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.25);
```
Should we register "shadows" as our next tokens to catalogue?